src/main/java/org/example/User2.java

```diff
@@ -0,0 +1,7 @@
+package org.example;
+
+public class User2 {
+    public static String username2;
+    public  static String password2;
+    boolean loggedIn;
+}
```

Comment:
Are you ABSOLUTLEY sure that's how you should handle multiple Users? Did you know that Facebook had over 3 billion users?